src/libs/patches/make_spdlog_compile_linux.patch

```diff
@@ -0,0 +1,25 @@
+From c0fcb37692da01049b4acfd92a779738c14d32ff Mon Sep 17 00:00:00 2001
+From: Dan Crankshaw <dscrankshaw@gmail.com>
+Date: Thu, 2 Feb 2017 15:05:56 -0800
+Subject: [PATCH] make compile on alpine
+
+---
+ include/spdlog/details/os.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/include/spdlog/details/os.h b/include/spdlog/details/os.h
+index b63ce66..dae02f2 100644
+--- a/include/spdlog/details/os.h
++++ b/include/spdlog/details/os.h
```

Comment:
This is just the format for a git patch file (generated with `git format-patch`).